guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -147,16 +147,10 @@ endif::[]
 ----
 # {foreman-maintain} service restart
 ----
-. Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups that you make.
-. If you make changes in the previous step, restart the `{foreman-maintain}` services.
-+
-[options="nowrap" subs="attributes"]
-----
-# {foreman-maintain} service restart
-----
```

Comment:
This looks like an unrelated change.

Reply:
removed

Reply:
Am I wrong or does this PR still remove those lines, which I consider an unrelated change.

Reply:
I removed them on this PR, is that what you are confirming?

Reply:
It's part of "upgrading Foreman"; how does this relate to the pulp3 migration?

Reply:
@maximiliankolb the previous discussion on this question is here: https://github.com/theforeman/foreman-documentation/pull/707#discussion_r703647893

that discussion suggests that those changes were added in error, and I have therefore removed them, the same way they were removed in #707 

those lines, if they are indeed required, can be addressed in another issue and associated commit. 

but for now, let's merge this so we can release the beta docs in a matter of days, thank you.

Reply:
Just confirming that these changes are required. They are duplicative steps. Yes, they ought to be in their own PR, but Ian included them originally and I have done the same.

Duplication:
https://github.com/theforeman/foreman-documentation/blob/master/guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc